include/deal.II/grid/grid_generator.h

```diff
@@ -768,6 +768,64 @@ namespace GridGenerator
                       const Point<dim> &  center = Point<dim>(),
                       const double        radius = 1.);
 
+  /**
+   * Generate a 2D mesh consisting of the unit square joined with a copy shifted
+   * by $s = (1,0)$. Depending on the flags passed either the right or the left
+   * square is rotated by $\pi/2$. This way one can generate a mesh in which one
+   * square possibly contains an edge that has the opposite tangiantial (and
+   * hence also opposite normal) orientation of the neighboring edge of the
+   * other square.
+   *
+   * This mesh is not overly useful from the practical point of view. For
+   * debugging purposes it can be used to check for orientation issues for
+   * tensor valued finite elements.
+   *
+   * @note If <code>rotate_left_square==rotate_right_square</code> the mesh is consistently oriented.
+   *
+   * @tparam dim
```

Comment:
Just drop the line if you don't want to provide a discussion of what the parameter means (which is not necessary).